client.lua

```diff
@@ -481,6 +505,8 @@ RegisterNetEvent('qb-vehicleshop:client:openVehCats', function(data)
     lib.showContext('open_veh_cats')
 end)
 
+--- ?
+---@params data table[]
```

Comment:
data here is not an array. We know this because it has fields such as price, and buyVehicle. Therefore data is a table.